lib/webapp.js

```diff
@@ -2015,6 +2049,7 @@ WebApp.prototype = {
     }
     this._resolveImports(plugin, urlBase);
     this.plugins.push(plugin);
+    pluginsArray.push(plugin);
```

Comment:
Also I'm not seeing any major difference between `this.plugins` and `pluginsArray` am I missing something? One gets defined at the start of the file, the other gets defined in constructor. They both get pushed the same stuff & they don't get altered later in this file

Reply:
I have used `pluginsArray` because `this.plugins` is not accessible inside `call()`. Please let me know if I am missing something.

Reply:
I see 2 ways:

1. `WebApp `eventually creates `WebServiceHandle` 's as I understand, so `this.plugins` could be passed through any method that creates `WebServiceHandle`, to make maybe `WebServiceHandle.plugins` or
(better, easier?) 2. Make `this.plugins` a global object for the file, instead of a `WebApp` object